src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -195,6 +195,36 @@ fields specific to anatomical scans:
 | ----------------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
 | ContrastBolusIngredient | OPTIONAL              | [string][]    | Active ingredient of agent. Values MUST be one of: IODINE, GADOLINIUM, CARBON DIOXIDE, BARIUM, XENON Corresponds to DICOM Tag 0018,1048. |
 
+The [`part-<label>`](../99-appendices/09-entities.md#part) key/value pair is
+used to indicate which component of the complex representation of the MRI
+signal is represented in voxel data.
+This entity is associated with the DICOM tag `0008,9208`.
+Allowed label values for this entity are `phase`, `mag`, `real` and `imag`,
+which are typically used in `mag/phase` or `real/imag` pairs.
```

Comment:
>  which are typically used in `mag/phase` or `real/imag` pairs.

this may make some people believe they have to specify `*_part-mag/phase*`, whereas the example shows that it should be rather a pair of files - one for mag, one for phase (vice versa for real, imag).

Reply:
What about the following?

> which are typically used in `part-mag`/`part-phase` or `part-real`/`part-imag` pairs of files.

Reply:
Much better, thanks!